[2025-07-09T06:30:41.533+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T06:30:41.554+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_airflare_data.transform manual__2025-07-09T06:30:36.780848+00:00 map_index=3 [queued]>
[2025-07-09T06:30:41.561+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_airflare_data.transform manual__2025-07-09T06:30:36.780848+00:00 map_index=3 [queued]>
[2025-07-09T06:30:41.561+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-09T06:30:41.571+0000] {taskinstance.py:2330} INFO - Executing <Mapped(_PythonDecoratedOperator): transform> on 2025-07-09 06:30:36.780848+00:00
[2025-07-09T06:30:41.579+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5489) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-09T06:30:41.581+0000] {standard_task_runner.py:63} INFO - Started process 5496 to run task
[2025-07-09T06:30:41.581+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_airflare_data', 'transform', 'manual__2025-07-09T06:30:36.780848+00:00', '--job-id', '474', '--raw', '--subdir', 'DAGS_FOLDER/Untitled-1.py', '--cfg-path', '/tmp/tmptja4r0gn', '--map-index', '3']
[2025-07-09T06:30:41.582+0000] {standard_task_runner.py:91} INFO - Job 474: Subtask transform
[2025-07-09T06:30:41.643+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_airflare_data.transform manual__2025-07-09T06:30:36.780848+00:00 map_index=3 [running]> on host e6a32b72ede0
[2025-07-09T06:30:41.752+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_airflare_data' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T06:30:36.780848+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T06:30:36.780848+00:00'
[2025-07-09T06:30:41.754+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T06:30:41.754+0000] {Untitled-1.py:136} INFO - Данные приняты из Xcom, начат процесс очистки
[2025-07-09T06:30:41.755+0000] {Untitled-1.py:58} INFO - Создание подключения к S3
[2025-07-09T06:30:41.756+0000] {Untitled-1.py:62} INFO - Подключение установлено, загрузка данных из S3
[2025-07-09T06:30:41.763+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T06:30:41.764+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T06:30:41.895+0000] {Untitled-1.py:68} INFO - Данные выгружены из S3
[2025-07-09T06:30:41.900+0000] {Untitled-1.py:145} INFO - Преобразование завершено, начинается передача DF в S3
[2025-07-09T06:30:41.901+0000] {Untitled-1.py:26} INFO - Создание подключения к S3
[2025-07-09T06:30:41.902+0000] {Untitled-1.py:31} INFO - Подключение установлено
[2025-07-09T06:30:41.909+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T06:30:41.909+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T06:30:42.259+0000] {Untitled-1.py:37} INFO - Загрузка данных в S3
[2025-07-09T06:30:42.515+0000] {Untitled-1.py:48} INFO - Загрузка завершена успешно
[2025-07-09T06:30:42.516+0000] {Untitled-1.py:153} INFO - Данные успешно сохранены в S3, передаю их в Xcom
[2025-07-09T06:30:42.516+0000] {Untitled-1.py:157} INFO - данные успешно очищены и записаны в хранилище
[2025-07-09T06:30:42.516+0000] {python.py:237} INFO - Done. Returned value was: {'bucket': 'airflare', 'key': 'temp/df_HKT_IKT.json'}
[2025-07-09T06:30:42.517+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T06:30:42.535+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_airflare_data, task_id=transform, run_id=manual__2025-07-09T06:30:36.780848+00:00, map_index=3, execution_date=20250709T063036, start_date=20250709T063041, end_date=20250709T063042
[2025-07-09T06:30:42.597+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T06:30:42.618+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-09T06:30:42.619+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
